modules/openxAnalyticsAdapter.js

```diff
@@ -1,365 +1,90 @@
-import includes from 'core-js-pure/features/array/includes.js';
 import adapter from '../src/AnalyticsAdapter.js';
 import CONSTANTS from '../src/constants.json';
 import adapterManager from '../src/adapterManager.js';
-
-//* *******  V2 Code
 import { ajax } from '../src/ajax.js';
+const utils = require('../src/utils.js');
 
-// temp dependency on zlib to minimize payload
-const zlib = require('zlib');  // eslint-disable-line
+export const AUCTION_STATES = {
+  INIT: 'initialized', // auction has initialized
+  ENDED: 'ended', // all auction requests have been accounted for
+  COMPLETED: 'completed' // all slots have rendered
+};
 
-const utils = require('../src/utils.js');
 
-const urlParam = '';
-const analyticsType = 'endpoint';
 
 const ADAPTER_VERSION = '0.1';
 const SCHEMA_VERSION = '0.1';
 
-const MAX_RETRIES = 2;
-const MAX_TIMEOUT = 10000;
 const AUCTION_END_WAIT_TIME = 1000;
+const URL_PARAM = '';
+const ANALYTICS_TYPE = 'endpoint';
+const ENDPOINT = 'https://prebid.openx.net/ox/analytics';
 
-const auctionInitConst = CONSTANTS.EVENTS.AUCTION_INIT;
-const auctionEndConst = CONSTANTS.EVENTS.AUCTION_END;
-const bidWonConst = CONSTANTS.EVENTS.BID_WON;
-const bidRequestConst = CONSTANTS.EVENTS.BID_REQUESTED;
-const bidAdjustmentConst = CONSTANTS.EVENTS.BID_ADJUSTMENT;
-const bidResponseConst = CONSTANTS.EVENTS.BID_RESPONSE;
-const bidTimeoutConst = CONSTANTS.EVENTS.BID_TIMEOUT;
+// Event Types
+const {
+  EVENTS: { AUCTION_INIT, BID_REQUESTED, BID_RESPONSE, BID_TIMEOUT, AUCTION_END, BID_WON }
+} = CONSTANTS;
 const SLOT_LOADED = 'slotOnload';
 
+const UTM_TAGS = [
+  'utm_campaign',
+  'utm_source',
+  'utm_medium',
+  'utm_term',
+  'utm_content'
+];
+const UTM_TO_CAMPAIGN_PROPERTIES = {
+  'utm_campaign': 'name',
+  'utm_source': 'source',
+  'utm_medium': 'medium',
+  'utm_term': 'term',
+  'utm_content': 'content'
+};
+
 /**
- * @typedef {Object} AnalyticsConfig
+ * @typedef {Object} OxAnalyticsConfig
  * @property {string} publisherPlatformId
  * @property {number} publisherAccountId
  * @property {number} sampling
  * @property {boolean} enableV2
```

Comment:
is that still needed ?

Reply:
Being paranoid that they might typedef the common analytics config